src/components/diffViewer/DiffViewer.tsx

```diff
@@ -46,6 +47,6 @@ export class DiffViewer extends React.Component<DiffViewerProps> {
 
         return !DiffViewer.EmptyHtmlRegex.test(html)
             ? <div className={styles.diffViewer} dangerouslySetInnerHTML={{__html: html}}></div>
-            : <BlankSlate title={this.props.noChangesLabel} />;
+            : <BlankSlate title={this.props.noChangesLabel} description={this.props.noChangesDescription} />;

```

Comment:
Did you try with `className='mod-small'` ? The blankslate seems a little bit huge IMHO

Reply:
![screen shot 2018-08-07 at 10 54 43 am](https://user-images.githubusercontent.com/3683420/43783721-6719454e-9a30-11e8-8a84-ae575b161d1c.png)


VS.

![screen shot 2018-08-07 at 10 55 05 am](https://user-images.githubusercontent.com/3683420/43783731-6c544608-9a30-11e8-8463-3a189822b15c.png)

@otaillon do you have a preference?

Reply:
Second one is good ! More breathing room